pyproject.toml

```diff
@@ -14,5 +14,5 @@ keywords = "pythonic tk tcl tkinter gui beginner"
 [tool.flit.metadata.requires-extra]
 # lxml is in soup_viewer because examples/soup.py does BeautifulSoup(string, 'lxml')
 # exact svglib version because: https://github.com/Akuli/teek/issues/13
-image_loader = ["pillow", "reportlab", "svglib==0.9.0", "lxml"]
-soup_viewer = ["beautifulsoup4", "lxml"]
+image_loader = ["pillow", "reportlab", "svglib==0.9.0", "lxml==4.3.5"]
+soup_viewer = ["beautifulsoup4", "lxml==4.3.5"]
```

Comment:
Was this because of the Python 3.4 issue? I think it would be better to delete Python 3.4 from travis.yml, because nobody uses python 3.4 anymore.